server/mail/mail.go

```
@@ -38,20 +38,18 @@ func getMessageBody(e kolide.Email) ([]byte, error) {
 }
 
 func (dm devMailService) SendEmail(e kolide.Email) error {
-	if !e.Config.SMTPDisabled {
-		if e.Config.SMTPConfigured {
-			msg, err := getMessageBody(e)
-			if err != nil {
-				return err
-			}
-			fmt.Printf(string(msg))
+	if e.Config.SMTPEnabled && e.Config.SMTPConfigured {
+		msg, err := getMessageBody(e)
+		if err != nil {
+			return err
 		}
+		fmt.Printf(string(msg))
 	}
 	return nil
 }
 
 func (m mailService) SendEmail(e kolide.Email) error {
-	if !e.Config.SMTPDisabled && e.Config.SMTPConfigured {
+	if e.Config.SMTPEnabled && e.Config.SMTPConfigured {
```

I believe if SMTPEnabled is false, we should return an error. I tried this on master yesterday with email disabled, and was still able to send email. Obviously nothing happened(and there was no error) making the experience somewhat confusing.